engine/src/main/java/org/terasology/input/cameraTarget/TargetSystem.java

```diff
@@ -44,6 +44,15 @@ public TargetSystem(BlockEntityRegistry blockRegistry, Physics physics) {
         this.physics = physics;
     }
 
+    /**
+     * Gets the position of the block that is currently targeted. If there is currently no target, this will return
+     * a null reference (isTargetAvailable() would have returned false in this case).
+     * @return the target block position in world coordinates, a vector of 3 integers.
+     */
+    public Vector3i getTargetBlockPosition() {
```

Comment:
Tagging @Josharias: he might be more knowledgeable than I am about this part of the codebase.